.github/workflows/update_sdk.yml

```diff
@@ -0,0 +1,132 @@
+        $jsonPath = "win32generator\resources\metadata\versions.json"
+        $jsonContent = Get-Content -Path $jsonPath -Raw | ConvertFrom-Json
+        $version = $jsonContent."${{ matrix.sdk.name }}"
+        if ([System.Version]$env:latestVersion -le [System.Version]$version) {
+          Write-Output "There isn't any new version of ${{ matrix.sdk.name }}."
+          echo "isSuccess=false" >> $env:GITHUB_OUTPUT
+        } else {
+          echo "isSuccess=true" >> $env:GITHUB_OUTPUT
+        }
+      working-directory: win32more
+
+    - name: Checkout winmd-printer code
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      uses: actions/checkout@v4
+      with:
+        repository: ynkdir/winmd-printer
+        path: winmd-printer
+
+    - name: Setup .NET
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 8.x
+
+    - name: Setup Python
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.13"
+
+    - name: Setup 7-Zip
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: choco install 7zip --yes
+
+    - name: Run ${{ matrix.sdk.script_file_name }}
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: ${{ matrix.sdk.script_file_name }} $env:latestVersion
+      working-directory: winmd-printer
+
+    - name: Compress ${{ matrix.sdk.json_file_name }} with 7-Zip
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: |
+        Remove-Item win32more\win32generator\resources\metadata\${{ matrix.sdk.json_file_name }}.xz
+        7z a -txz win32more\win32generator\resources\metadata\${{ matrix.sdk.json_file_name }}.xz winmd-printer\${{ matrix.sdk.json_file_name }}
+
+    - name: Run win32generator
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: python -m win32generator
+      working-directory: win32more
+
+    - name: Update versions.json
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: |
+        $jsonPath = "win32generator\resources\metadata\versions.json"
+        $jsonContent = Get-Content -Path $jsonPath -Raw | ConvertFrom-Json
+        $jsonContent."${{ matrix.sdk.name }}" = $env:latestVersion
+        $updatedJson = $jsonContent | ConvertTo-Json
+        Set-Content $jsonPath $updatedJson
+      working-directory: win32more
+
+    - name: Commit changes
+      if: ${{ steps.is-there-update.outputs.isSuccess == 'true' }}
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "github-actions[bot]@users.noreply.github.com"
+        git add .
+        git commit --allow-empty -m "${{ matrix.sdk.commit_message }} $env:latestVersion"
+        git push origin "${{ github.event.repository.default_branch }}"
+      working-directory: win32more
```

Comment:
Is it possible to make pull request?

Reply:
Fixed in https://github.com/ynkdir/py-win32more/pull/66/commits/bed5634d6833af9d9d2ac7fcc52f41977fae8404